### PR TITLE
Search for [^A-Za-z0-9] instead of the unsupported \W

### PR DESCRIPTION
**Pull request by iBug**
MS has commented a lot of "0 TPs, 0 FPs", in Smokey's PRs. This should fix that wrong search.

P.S. This is my *absolutely first* lines of Ruby code so there may be very stupid syntax or usage errors. Please check them, Thanks.

### CODE REVIEW

**Comment by iBug** [comment]
I'm not sure what code I wrote actually. I hope a Ruby developer understand my intention. lol.

File: app/controllers/github_controller.rb [hunk 1]
```diff
@@ -85,6 +85,7 @@ def pull_request_hook
 
     keywords.each do |keyword|
       keyword = keyword[0]
+      keyword.gsub! '\W', '[ -]'
```
[the remark this answers is not in the document]
Comment (ArtOfCode-) [reply app/controllers/github_controller.rb]: Add an underscore to that class, but yeah

**Comment by iBug** [comment]
@ArtOfCode- @j-f1 Yep. Changes made.

**Comment by ArtOfCode-** [comment]
Note: I do intend to review this, but a scroll through the changes tab doesn't give me confidence it's going to work... search is complicated and I'd like to actually pull and test locally before I give this a pass.

**Comment by j-f1** [comment]
Does the `LIKE` → `REGEXP` change add support for `\W`?

**Comment by iBug** [comment]
@j-f1 I'm afraid it's because [MySQL does not support `\W`][1] at all...

  [1]: https://stackoverflow.com/posts/comments/10144453

**Pull request merged**